lib/reverse_proxy_plug.ex

```diff
@@ -359,7 +364,7 @@ defmodule ReverseProxyPlug do
   end
 
   defp get_cookies(%Conn{req_cookies: cookies}) do
-    cookies |> Enum.map(fn {k, v} -> "#{k}=#{v}" end) |> Enum.join("; ")
+    cookies |> Enum.map_join("; ", fn {k, v} -> "#{k}=#{v}" end)
```

Comment:
Given this is a conceptually separate change from `host` - I factored this (and the `credo` update) into https://github.com/tallarium/reverse_proxy_plug/pull/160